backend/apis/clients/get-client.api.js

```diff
@@ -188,3 +188,164 @@ function getClientById(clientId, cbk, connection) {
     cbk(err, client);
   });
 }
+
+exports.getAllClientsById = getAllClientsById;
```

Comment:
I'm excited about this new function. Let's make the following changes:

1. Create a new file for it - `get-clients-by-id.api.js`
2. Expose it as an API so the browser can also call it
3. Document the API, maybe by adding a new section to https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/get-client.md